src/autoyast-rnc/kdump.rnc

```diff
@@ -41,10 +41,8 @@ kdump_general = element general {
     element KDUMP_REQUIRED_PROGRAMS { STRING }? &
     element KDUMP_PRESCRIPT { STRING }? &
     element KDUMP_POSTSCRIPT { STRING }? &
-    element KDUMPTOOL_FLAGS { STRING }? &
     element KDUMP_NETCONFIG { STRING }? &
     element KDUMP_NET_TIMEOUT { STRING }? &
-    element KDUMP_COPY_KERNEL { STRING }? &
```

Comment:
Is it OK to drop some keys in the AutoYaST profile? That means the old profiles won't validate... Maybe just ignoring them in the code is enough?

Reply:
If the option appears in /etc/sysconfig/kdump, kdump will just print out a warning that the option is deprecated, it will not break anything. So if it's a problem for YaST to not write this into the config file, it's not a big problem.

Reply:
Important is also the AutoYaST POV, I'm not sure how we should behave regarding the old XML profiles.

Either we silently ignore the old options or we let the validation fail. I'm not sure what is better..

@imobachgs ?